### ISSUE TITLE
This unfortunately does not work for me

### ISSUE THREAD

**Issue by Gunhawk007** [Flawioo/Creality-Ender-3-board-v4.2.7#13]
Hi,
I am really struggling with the Marlin FW for this board. Although I could run the stock FW that Creality offers in compiled form on their website, I could not get my own FW build to work. The printer always starts with a blank screen after trying to read the FW from the SD-Card. I should have same Hardware config as I do have a stock Ender 3 Pro, but upgraded it with Crealities 4.2.7 Silent Board with TMC 2225 Steppers and have a BLTouch 3.1 probe installed. 
I even have tried to get your compiled binary to work but same result as it is for my own builds (blank screen) so I am really lost. I don't think there is noone on the planet with my configuration that seem to be a pretty common upgrade that have their own Marlin FW running

**Comment by Flawioo** [comment]
Hi!

What kind of display are you using?

**Comment by Gunhawk007** [comment]
I am using the LCD screen that kame with the stock Ender 3 Pro. I am aware that the Ender 3 V2 has a different one but my machine uses the old one.

**Comment by Flawioo** [comment]
Sounds ok about the LCD setup to me.

Have you already tried with another board. I think you had some bad luck with your motherboard my friend.

It's not a firmware issue, many many guys have had good results with precompiled firmwares.

Let us know any news ok.

**Comment by Gunhawk007** [comment]
Unfortunately I do not have access to a spare mainbord to test. 

I have opened a support case with creality indicating a hardware problem.

 Meanwhile I was able to revive my printer with another firmware file I found on the Internet. I sent it to them and they confirmed to me this being identical to the one I tried from their website that was giving me the blank screen issue. 
 
So at least I do have a board that has unreliable behavior updating the firmware. 

I am trying to update because I want to install my filament sensor in addition to the BLTouch and according to Creality there is no official firmware out to support both - at least not for the new boards that came without the transfer board - the firmware posted on their website that support both features is only for the boards with the transfer board.

In addition to that in my current firmware I could not do fine adjustments to the Z-Offset for bed leveling. It only allow for increments of 0.1 mm -unlike the Move Axis command that allow for fine adjustments up to 0.025mm. Therefor I suspect that they have no babystepping enabled in their firmware.

I keep you updated how they respond.
